Add Clerk CSP guide (#1076)
clerkinc/clerk-docs: docs/security/clerk-csp.mdx

---
description: Learn how to configure your Content Security Policy to work with Clerk's APIs.
---

# Configure Clerk Content-Security-Policy headers

[Content-Security-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) (CSP) headers secure your document by preventing resources from being loaded from unexpected sources. This protects your apps from [XSS](/docs/security/xss-leak-protection#xss-leak-protection) attacks and data injections.

You can configure the following CSP headers:

> 1. `script-src` – This value should be the host application's FAPI hostname, for example `https://clerk.your-domain.com`.

Reviewer: I think this reads a little better

```suggestion
1. `script-src` – This value should be the host application's FAPI hostname, such as `https://clerk.your-domain.com`.
```

or if you wanted to keep "for example":

```suggestion
1. `script-src` – This value should be the host application's FAPI hostname, for example: `https://clerk.your-domain.com`.
```

please apply whichever to the other bullet points that do this as well !
